aya/src/programs/xdp.rs

```diff
@@ -209,6 +224,7 @@ impl Xdp {
     }
 }
 
+// TODO (AM): implement Drop to force detach?
```

Comment:
perhaps a dumb question but is there a reason we don't implement `Drop` on `NdLink` to force detach? I think that'd simplify some of the stuff that wrap `NlLink` that have to remember to call for `detach` manually during drop and optionally `forget`.